### ISSUE TITLE
Focus indicator on the legend is overlapping with it’s label and legend text truncated at 400% zoom.

### ISSUE THREAD

**Issue by msftedad** [highcharts/highcharts#19459]
#### Expected behaviour
Focus indicator should be clearly visible on the single legend when it receives keyboard focus, and the legends text should get truncated at 400% zoom.

#### Actual behaviour
Focus indicator is not properly visible on the single legend when it receives keyboard focus, Here the keyboard focus indicator is overlap with the legends text.

#### Live demo with steps to reproduce

1. Open URL: [A Pen by Elvis (codepen.io)](https://nam06.safelinks.protection.outlook.com/?url=https:%2f%2fcodepen.io%2flydoc1%2fpen%2frNQYEvL&data=05%7c01%7cv-hemalathav%40microsoft.com%7c9ce42a5925cf4ff610ff08db8d11a1bf%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c638258879109757904%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c3000%7c%7c%7c&sdata=ccW9LnsH%2fnslmGA7NGbkPYnv7SOKDm5fwqO%2fswF5B0U%3D&reserved=0)
2. Navigate to legends present at line graph.
3. Apply settings.... Settings -> Display -> Display resolution -> Set display at 1280*1024.
4. Observe the issue whether the focus indicator is overlaps with its label or not.

#### Product version

Version-11.1.0

#### Affected browser(s)
Windows 11 Enterprise
OS: 22H2
Verion: 22621.1992
Browsers: Edge Version 115.0.1901.188 (Official build) (64-bit)

**Attachments added:**
<img width="615" alt="Focus indicator is overlaps with legend" src="https://github.com/highcharts/highcharts/assets/93735775/edc90e85-f2aa-4e2e-bd43-1be91c06487e">

https://github.com/highcharts/highcharts/assets/93735775/110ddb33-2511-4c75-9a7e-f3e2174c3621

**Comment by bm64** [comment]
Hello @msftedad and thanks for reporting the issue.

I'm closing this ticket as a duplicate of https://github.com/highcharts/highcharts/issues/19335
Feel free to continue discussion in the original topic
